pages/bib/[id].tsx

```diff
@@ -53,6 +56,7 @@ export default function BibPage({
   const { pathname, push, query } = useRouter()
   const metadataTitle = `Item Details | ${SITE_NAME}`
   const bib = new Bib(discoveryBibResult)
+  const displayCatalogLink = !!bib.id.startsWith("b")
```

Comment:
can you use this [`isNyplOwned` util ](https://github.com/NYPL/research-catalog/blob/main/src/utils/bibUtils.ts#L59)here? And can the variable say displayLegacyCatalogLink?